addons/web_unsplash/views/res_config_settings_view.xml

```diff
@@ -18,6 +18,7 @@
                     </div>
                 </div>
             </div>
+            <div id="web_unsplash_warning" position="replace"/>
```

Comment:
That line is not required if you do it directly in existing xpath.
Could you please instead do
```xml
<div id="web_unsplash_warning" position="replace">
```
instead of
```xml
<div id="web_unsplash_settings" position="inside">
```
a few line above and just remove this line.